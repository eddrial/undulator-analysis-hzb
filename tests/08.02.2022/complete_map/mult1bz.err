 error report for the data set mult1bz.dvm         
--------------------------------------------
      
         127   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2558 data points
 the derivative at  0.43190E+03 and  0.43782E+03 is zero 
 the derivative at  0.43782E+03 and  0.44162E+03 is zero 
 the derivative at  0.60683E+03 and  0.63852E+03 is zero 
 the derivative at  0.66387E+03 and  0.70063E+03 is zero 
 the derivative at  0.70063E+03 and  0.72218E+03 is zero 
 the derivative at  0.10260E+04 and  0.10572E+04 is zero 
 the derivative at  0.10572E+04 and  0.10881E+04 is zero 
 the derivative at  0.11143E+04 and  0.11384E+04 is zero 
 the derivative at  0.12753E+04 and  0.13065E+04 is zero 
 the derivative at  0.14739E+04 and  0.14971E+04 is zero 
 the derivative at  0.14971E+04 and  0.15326E+04 is zero 
 the derivative at  0.16137E+04 and  0.16454E+04 is zero 
 the derivative at  0.17832E+04 and  0.18081E+04 is zero 
 the derivative at  0.18668E+04 and  0.18901E+04 is zero 
 the derivative at  0.18901E+04 and  0.18909E+04 is zero 
 the derivative at  0.18909E+04 and  0.18913E+04 is zero 
 the derivative at  0.18913E+04 and  0.19222E+04 is zero 
 the derivative at  0.20908E+04 and  0.20988E+04 is zero 
 the derivative at  0.20988E+04 and  0.21030E+04 is zero 
 the derivative at  0.21030E+04 and  0.21165E+04 is zero 
 the rms value of the residuals is  0.20435E-02
 the maximum residual is  0.71968E-02
 the fitted period length is  0.55779E+02
 the rms value of the lambda/2-filtered data is  0.12426E-02
 the maximum value of the lambda/2-filtered data is  0.49738E-02
