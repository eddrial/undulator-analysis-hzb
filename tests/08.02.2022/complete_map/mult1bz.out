mult1bz.dvm 
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  7.7596074E-03
   2  1.4802995E-04
   3 -1.0337494E-04
   4  7.3418619E-06
   5  9.2593891E-06
   6  5.5778812E+01
   7  1.1954386E+03
   8  1.1056270E+03
   9  1.0434210E+03
  10 -2.4063303E-01
  11  7.0112816E+01
 number of iterations:   14
     sumsq:  6.64406E-03
       rms:  2.04353E-03  dimension of magnetic field
       rms:  2.63349E+01  percent of b_eff
   rms_ext:  2.38404E+01  percent of b_eff
 B_eff =   7.7598E-03
 B_max =   7.9022E-03
 K_eff =    0.041
 lamda_1 (nm) =      2.54237

        eigenvectors in parameter space

 error                         direction
    12.56,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     7.75,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     1.45,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.02  0.00  0.00
     1.25,  0.00  0.00  0.00  0.00  0.00  0.00  0.00 -0.02  1.00  0.00  0.00
     0.08,  0.00  0.00  0.00  0.00  0.00 -0.02  1.00  0.00  0.00  0.00  0.00
     0.01,  0.00  0.00  0.00  0.00  0.00  1.00  0.02  0.00  0.00  0.00  0.00
     0.00, -0.56  0.74 -0.06  0.34  0.17  0.00  0.00  0.00  0.00  0.00  0.00
     0.00, -0.01 -0.27  0.71  0.58  0.28  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.26  0.00 -0.30 -0.07  0.92  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.51  0.08 -0.44  0.70 -0.23  0.00  0.00  0.00  0.00  0.00  0.00
     0.00,  0.60  0.61  0.45 -0.24 -0.04  0.00  0.00  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.06803 Tmm
 kick_2:   0.06803 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.14343E+02
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.20435E-02
 the maximum residual is  0.71968E-02
      
 the rms value of the lambda/2-filtered data is  0.12426E-02
 the maximum value of the lambda/2-filtered data is  0.49738E-02
      
 the rms phase error (all data) is    2.923 deg.
 the maximum phase error (all data) is    8.033 deg.
      
 the rms phase error (extrema) is   12.212 deg.
 the maximum phase error (extrema) is   27.718 deg.
      
 the Walker-rms phase error (all data) is    2.922 deg.
 the Walker-maximum phase error (all data) is    8.004 deg.
      
 the Walker-rms phase error (extrema) is    2.991 deg.
 the Walker-maximum phase error (extrema) is    7.837 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             2.922570    
 local phase error             1.177698    
 slowly varying phase error  NaN           
 oscillatory phase error     NaN           
 -----------------------------------------
